openstack/helpers/projectpurge.go

```diff
@@ -229,8 +258,14 @@ func clearNetworkingPorts(projectID string, networkClient *gophercloud.ServiceCl
 	}
 	if len(allPorts) > 0 {
 		for _, port := range allPorts {
+			if port.DeviceOwner == "network:ha_router_replicated_interface" {
+				fmt.Printf("Skipping port: %s with owner: %s\n", port.ID, port.DeviceOwner)
+				continue
+			}
+
 			err = ports.Delete(networkClient, port.ID).ExtractErr()
 			if err != nil {
+				fmt.Println(err)
```

Comment:
I think we can remove this?

Reply:
Ups, yes these are some debugging statements. Are there any plans to support a logging framework e.g. it could be useful to see which resources are deleted.

Reply:
I think if we were to add logging, it would be in the form of a logger interface like here: https://github.com/gophercloud/utils/blob/master/client/client.go

This way we don't have to make a choice of logger that some people would prefer not to use.